packages/manager/modules/pci/src/projects/project/serving/namespace/models/models.html

```diff
@@ -0,0 +1,79 @@
+<div>
+    <p class="mb-3">
+        <span data-translate="pci_projects_project_serving_namespace_models_description_1"
+              data-translate-values="{
+                  description: $ctrl.project.description
+              }">
+        </span>
+        <span data-translate="pci_projects_project_serving_namespace_models_description_2"></span>
+    </p>
+    <oui-datagrid data-rows="$ctrl.models" data-row-loader="$row">
```

Comment:
```suggestion
    <oui-datagrid data-rows="$ctrl.models">
```
Not needed if you already have all the needed data